### PR TITLE
Grammatical errors of README-wsrep fixed.

### PR DESCRIPTION
**Pull request by rakshitkumarcse**

### CODE REVIEW

**Comment by rakshitkumarcse** [comment]
@an3l @svoj  Can you also tell me about how I can contribute to MariaDB properly? I have been using mysql for a sufficiently long time and consider myself skilled enough to fix bugs and issues if guided. Kindly guide me in the right direction. Also, I am aiming to take a project under MariaDB in GSOC 2k19. It would be great if you could help me in this regard.

**Comment by janlindstrom** [comment]
Thank you for your contribution.

Similar to other open source projects, the MariaDB Foundation needs to have shar
ed ownership of all code that is included in the MariaDB distribution. The easie
st way to achieve this is by submitting your code under the BSD-new license.

The other alternative is to sign the code contribution agreement which can be fo
und here: https://mariadb.com/kb/en/mariadb/mca/

Please indicate in a comment below that you are contributing your new code of th
e whole pull request, including one or several files that are either new files o
r modified ones, under the BSD-new license or that you have filled out the contr
ibution agreement and sent it.

Thanks,
Jan

**Comment by janlindstrom** [comment]
In my understanding we need the license for every PR or MCA covering all.

**Comment by rakshitkumarcse** [comment]
I hereby accept and submit this code/PR under the BSD license.

File: Docs/README-wsrep [hunk 1]
```diff
@@ -266,8 +266,7 @@ binlog_format=ROW
 innodb_autoinc_lock_mode=2
    This is a required parameter. Without it INSERTs into tables with
    AUTO_INCREMENT column may fail.
-   autoinc lock modes 0 and 1 can cause unresolved deadlock, and make
-   system unresponsive.
+   autoinc lock modes 0 and 1 can cause unresolved deadlock, and make the system unresponsive.
```
Comment (iangilfillan) [inline Docs/README-wsrep]: Maintain the line wrapping as it was before

File: Docs/README-wsrep [hunk 2]
```diff
@@ -412,8 +411,8 @@ wsrep_sst_auth=
 
 wsrep_sst_donor=
    A name of the node which should serve as state snapshot donor. This allows
-   to control which node will serve state snapshot request. By default the
-   most suitable node is chosen by wsrep provider. This is the same as given in
+controlling which node will serve state snapshot request. By default the
```
Comment (iangilfillan) [inline Docs/README-wsrep]: 'the' should be added here too, so "serve the state snapshop request"

File: Docs/README-wsrep [hunk 3]
```diff
@@ -412,8 +412,8 @@ wsrep_sst_auth=
 
 wsrep_sst_donor=
    A name of the node which should serve as state snapshot donor. This allows
-   to control which node will serve state snapshot request. By default the
-   most suitable node is chosen by wsrep provider. This is the same as given in
+controlling which node will serve the state snapshot request. By default the
```
Comment (svoj) [inline Docs/README-wsrep]: Please restore indentation here.

Comment (rakshitkumarcse) [reply Docs/README-wsrep]: @svoj done!
Thanks!

**Comment by svoj** [comment]
Thanks! Somehow I missed your question. I'd suggest you to browse jira.mariadb.org to look for feature requests and bugs. Take whatever you like, but post a comment that you're willing to work on it (so that there's no concurrent attempt to work on the same issue).

**Comment by an3l** [comment]
@rakshitkumarcse you can also see the `CONTRIBUTING.md` file `10.4` to take future direction of your contributions. 
Thanks

**Comment by rakshitkumarcse** [comment]
@an3l Thanks! Will start making PRs on technical issues hereforth.

**Comment by rakshitkumarcse** [comment]
@svoj Thanks a lot!

**Pull request merged**